conans/client/tools/files.py

```diff
@@ -55,7 +55,8 @@ def human_size(size_bytes):
     return "%s%s" % (formatted_size, suffix)
 
 
-def unzip(filename, destination=".", keep_permissions=False, pattern=None, output=None):
+def unzip(filename, destination=".", keep_permissions=False, pattern=None, output=None,
+          flat_folder=False):
```

Comment:
flat_folder name could be improved?, I would read it as getting rid of all levels of subfolders, not just dropping the first level one

Reply:
sure, suggestions? `crop_first_folder`, `flat_first`....?

Reply:
`ignore_root`?

Reply:
`skip_root`?

Reply:
`flat_root`?

Reply:
https://unix.stackexchange.com/questions/535772/what-does-strip-components-c-mean-in-tar

Probably ``strip_root``